### PR TITLE
[DOCU-1559][DOCU-1509] Global plugins in Konnect

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
Introducing shared config and global plugins, and doing some minor cleanup and adjustments to plugin info for ServiceHub.
* Shared config landing page: not a whole lot here right now, and not a whole lot to say. Not sure how to expand this at the moment, but it's out of scope for this PR.
* Global and consumer-scoped plugin config: generic instructions. Need to think about whether this is the best approach.
* ServiceHub plugins: update list of unsupported plugins (it's significantly shorter now) and move page to index - was getting feedback that people couldn't find the info as it was buried on the ServiceHub overview doc.

### Reason
https://konghq.atlassian.net/browse/DOCU-1509
https://konghq.atlassian.net/browse/DOCU-1559

### Testing
[Shared config page](https://deploy-preview-2986--kongdocs.netlify.app/konnect/shared-config/)
[Shared config: plugins](https://deploy-preview-2986--kongdocs.netlify.app/konnect/manage-plugins/shared-config/)
[Manage plugins: overview page](https://deploy-preview-2986--kongdocs.netlify.app/konnect/manage-plugins/)

### CODE REVIEW

File: app/konnect/manage-plugins/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+### Functionality differences from self-managed Kong Gateway
+
+The Portal Application Registration plugin is enabled automatically when you
+[enable application registration](/konnect/dev-portal/administrators/app-registration/enable-app-reg)
+for the Dev Portal. You don't need to enable it directly.
+
+Conversely, don't delete the Application Registration plugin directly.
+[Disable application registration](/konnect/dev-portal/administrators/app-registration/disable-app-reg)
+from the service overview only.
+
```
Comment (skibum) [inline app/konnect/manage-plugins/index.md:35]: This is not actually correct. We don't even use the Application Registration plugin in Konnect Cloud. We simply enabled the ACL and auth plugins (either key auth or OIDC) behind the scenes and then don't allow customer to modify the settings of those two plugins.

File: app/konnect/manage-plugins/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Apache OpenWhisk
+* Vault Auth
+
+See the [plugin compatibility chart](/konnect-platform/compatibility/plugins)
+for a full comparison of plans and network configurations that each plugin
+supports.
+
+## Custom plugins and serverless functions
+
+Currently, there is no way to add a custom plugin or a serverless function
```
Comment (skibum) [inline app/konnect/manage-plugins/index.md]: this is just the custom plugins at this point

**Review by skibum — APPROVED** [review]
Approved with minor comments

File: app/konnect/manage-plugins/index.md (hunk 2 above)
Comment (lena-larionova) [reply app/konnect/manage-plugins/index.md]: Fixed in https://github.com/Kong/docs.konghq.com/pull/2986/commits/91f4e71f69afa1de04dead7adc61926eb60d70f0.

**Review by Bradamant3 — APPROVED** [review]
Still feels a bit too inside itself -- that is, if you landed on one of these pages from search and you'd never used Gateway, you'd be pretty lost. But ... it's also the next iteration, not the final, so I'm disinclined to poke at it.

**Pull request merged**